tests/utils.js

```diff
@@ -261,8 +262,11 @@ const revertDb = async (except, ignoreRefresh) => {
   const needsRefresh = await revertSettings();
   await deleteAll(except);
   await revertTranslations();
+  if(ignoreRefresh){
+    await setUserContactDoc();
```

Comment:
Umm setting the userContactDoc doesn't make a lot of sense here. How come this is getting unset, or needs to be set? These docs should be always skipped on the "mass" deletion.